File: MATLAB/ASCII_30_Add.m
function y = ASCII_30_Add(x)
% For use on the German alphabet.  Temporarily append extended ASCII values to the end
% of the lower case character range
    n = size(x, 2);
    for i = 1 : n
        if (x(i) == 228) 
            y(i) = 123;
        elseif (x(i) == 246)
            y(i) = 124;
        elseif (x(i) == 252)
            y(i) = 125;
        elseif (x(i) == 223)
            y(i) = 126;
        else
            y(i) = x(i);
        end
    end
end

File: Julia/ASCII_30_Add.jl
function ASCII_30_Add(plaintext_ascii)
    plaintext_ascii[find(x -> x == 228, plaintext_ascii)] = 123
    plaintext_ascii[find(x -> x == 246, plaintext_ascii)] = 124
    plaintext_ascii[find(x -> x == 252, plaintext_ascii)] = 125
    plaintext_ascii[find(x -> x == 223, plaintext_ascii)] = 126
    return plaintext_ascii
end